process-migration-service/frontend/src/component/tabMigrationPlan/MigrationPlansBase.js

```diff
@@ -24,10 +24,17 @@ export default class MigrationPlansBase extends React.Component {
 
   retrieveAllPlans = () => {
     const servicesUrl = BACKEND_URL + "/plans";
-    axios.get(servicesUrl, {}).then(res => {
-      const plans = res.data;
-      this.setState({ plans, filteredPlans: plans });
-    });
+    axios
+      .get(servicesUrl, {})
+      .then(res => {
+        const plans = res.data;
+        this.setState({ plans, filteredPlans: plans });
+      })
+      .catch(() => {
```

Comment:
there's no communication with kieserver in this function, the catch block does not apply to kieserver failure.